### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by tsundvoll**

### CODE REVIEW

**Review by GodVenn — APPROVED** [review]
LGTM

File: backend/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ See [Using the “--env-file” option](https://docs.docker.com/compose/environm
 To run the backend in docker, run the following command in the root folder of flotilla:
 
 ```
-docker compose up backend --build
+docker-compose up --build backend
```
Comment (oysand) [inline backend/README.md:52]: Whether we use docker compose or docker-compose depends on if we have a newer version of docker compose

Comment (tsundvoll) [reply backend/README.md:52]: Ah, I didn't know. I see we are using the `docker-compose` format in the front-page README of flotilla, so at least we should be consistent here. Any thoughts on which format we should choose?

Comment (tsundvoll) [reply backend/README.md:52]: Let's go for the old format for now, and we can update then all later if we want.

**Pull request merged**